client/main.coffee

```diff
@@ -945,6 +945,18 @@ pointerEvents = ->
       ## Prevent right click from bringing up context menu, as it interferes
       ## with e.g. drawing.
       e.preventDefault()
+    wheel: (e) ->
+      e.preventDefault()
+      if e.ctrlKey
+        # Ideally this should zoom while keeping the point under the cursor centered.
+        # Instead, we zoom on the center of the window
```

Comment:
I agree. I'd really like to implement the ideal form, by generalizing `setScale` to zoom relative to a specified point, defaulting to the center; I don't think this should be hard.

Reply:
Implemented with new function `Board.setScaleHoldingOffsetCoordFixed: (newScale, point)`.